scipy/sparse/linalg/interface.py

```diff
@@ -459,13 +504,15 @@ def _transpose(self):
 class _CustomLinearOperator(LinearOperator):
     """Linear operator defined in terms of user-specified operations."""
 
-    def __init__(self, shape, matvec, rmatvec=None, matmat=None, dtype=None):
+    def __init__(self, shape, matvec, rmatvec=None, matmat=None,
+                 rmatmat=None, dtype=None):
```

Comment:
`rmatmat`  should go after `dtype` to retain backward compatibility (also in docstrign of LinearOperator).
Typically these will be used as keyword arguments so there's little usability drawback to maintaining strict backward compatibility.

Reply:
done